src/table/src/TableVirtualBody.js

```diff
@@ -46,7 +46,20 @@ export default class TableVirtualBody extends PureComponent {
      * When allowAutoHeight is true and this prop is true, the estimated height
      * will be computed based on the average height of auto height rows.
      */
-    useAverageAutoHeightEstimation: PropTypes.bool
+    useAverageAutoHeightEstimation: PropTypes.bool,
+
+    /**
+     * The scrollToIndex property passed to react-tiny-virtual-list
+     */
+    scrollToIndex: PropTypes.number,
+    /**
+     * The scrollOffset property passed to react-tiny-virtual-list
+     */
+    scrollOffset: PropTypes.number,
+    /**
+     * The scrollToAlignment property passed to react-tiny-virtual-list
+     */
+    scrollToAlignment: PropTypes.oneOf(['start', 'center', 'end', 'auto'])
```

Comment:
Since you are just passing props to another lib it's probably ok to omit the storybook, but maybe since we are only supporting these specific props it would be useful to add those to the docs and explain why you might need them (why we're exposing them)

Reply:
Added to the doc and to the example as inputs to control it

Reply:
Great! Thank you for doing that. Would you mind posting a screenshot or gif of the new addition to the docs?